test/upgradeableProxy/DispatcherRC4.upgrade.t.sol

```diff
@@ -35,15 +25,43 @@
 import {IUniversalChannelHandler} from "../../contracts/interfaces/IUniversalChannelHandler.sol";
 import {VirtualChain} from "../VirtualChain.sol";
 import {GeneralMiddleware} from "../../contracts/base/GeneralMiddleware.sol";
+import {Mars as MarsRc4, IbcDispatcher as IbcDispatcherRc4} from "./upgrades/MarsRc4.sol";
+import {
+    Earth as EarthRc4,
+    UniversalPacket as UniversalPacketRc4,
+    AckPacket as AckPacketRc4,
+    IbcUtils
+} from "./upgrades/EarthRc4.sol";
+
+abstract contract DispatcherRC4TestUtils is ChannelHandShakeUpgradeUtil {
+    function sendOnePacketRc4(bytes32 channelId, uint64 packetSeq, MarsRc4 sender) public {
+        vm.expectEmit(true, true, true, true);
+        emit SendPacket(address(sender), channelId, packet, packetSeq, timeoutTimestamp);
```

Comment:
what is `packet` actually set to here?

Reply:
packet is set from the inherited contract; [here](https://github.com/open-ibc/vibc-core-smart-contracts/blob/6f40816d6fa9f4572868a263b42a075323552b38/test/upgradeableProxy/Dispatcher.upgrade.t.sol#L86) it is set to the encoded string "msgPayload" 

I updated this lineuto be the hardcoded string in case that's better testing?